pkg/cli/admin/release/extract_tools.go

```diff
@@ -513,9 +514,23 @@ func (o *ExtractOptions) extractCommand(command string) error {
 				continue
 			}
 		}
+
+		if target.Arch == targetReleaseArch {
+			targetArchCommands[target.Command] = struct{}{}
+		}
+
 		if target.OS == "linux" && target.Arch == releaseArch {
-			klog.V(2).Infof("Skipping duplicate %s", target.ArchiveFormat)
-			continue
+			if _, ok := targetArchCommands[target.Command]; ok {
+				// Some target commands have release-arch types that defines extracting
```

Comment:
Was this corner case introduced with rhel8/rhel9 change? Or, has it been around for some time already?

Reply:
It has existed for a long time (from this commit https://github.com/openshift/oc/commit/3dba61ad7f65fa7d7945fb6bd1ba09c13cd88f06), but it didn't cause any problem because all the targets have `release-arch` type until now. oc.rhel9 and oc.rhel8 have their own static arch types.